src/core/diags/README.md

```diff
@@ -123,7 +121,7 @@ Get the tx power value(dBm) for diagnostics module.
 
 ```bash
 > diag power
-tx power: -10 dBm
+-10 dBm
```

Comment:
is `-10` more friendly for parser, also more consistent with its input.
```suggestion
-10
```

Reply:
Updated